src/app/routes/issues.new.tsx

```diff
@@ -103,6 +103,25 @@ export async function action({ request }: ActionFunctionArgs) {
                         )
                         .catch(console.error);
 
+                    await sendDiscordWebhookWithUrl(config.discord.studentServerIssueWebhookUrl, {
+                        thread_name: `${data.title} - #${issue.id}`,
+                        embeds: [
+                            {
+                                color: 0x22c55e,
+                                description: `A new [Issue](${config.baseUrl}/issues/${issue.id}) has been opened on the website.`,
```

Comment:
- Add title - looks a bit nicer.
- Make the whole message clickable - especially on mobile it's just more user friendly if the button to get to the issue is just bigger and not just on the very left side of the message.
```suggestion
                                title: 'New Issue',
                                description: `[A new Issue has been opened on the Student Council Website.](${config.baseUrl}/issues/${issue.id})`,
```